### PR TITLE
Adapters quality check template

### PR DESCRIPTION
**Pull request by philecs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,10 +20,17 @@
 
 | Workgroup  | Member Name | Role |
 | ------------- | ------------- | ------------- |
+<<<<<<< HEAD
```
Comment (jhorbulyk) [inline README.md]: Git Merge conflict not properly resolved.

File: Adapters/OpenQuestions.md [hunk 2]
```diff
@@ -27,31 +24,13 @@ adapter.
...
-* Cons: Two way linking becomes complicated.  One or both systems will have to
- support the capability to add additional fields.
-
-## Third Party System Which Links IDs
-* Incoming object already is aware of the id in `this` system because
- this information was previously fetched from a master data system.
-  * See **OIH** for implementation.
+from some  `other` system to the ids which exist in `this` system.  
+
+The ID linking is solved by using **OIHApplicationDataRecords**.
```
Comment (jhorbulyk) [inline Adapters/OpenQuestions.md]: Is this the result of some other workgroup?

Comment (philecs) [reply Adapters/OpenQuestions.md]: Yes it is from the Data & Domain Model Workgroup. See: [Global Rules and Regulations for OMDMs](https://github.com/openintegrationhub/Data-and-Domain-Models/tree/master/MasterDataModels#4-global-rules-and-regulations-for-omdms)

Comment (jhorbulyk) [reply Adapters/OpenQuestions.md]: Can you add the link to the markdown file?

Comment (philecs) [reply Adapters/OpenQuestions.md]: Sure. But I will only work for those who are part of this project. All other visitors won't be able to see this markdown file.

Comment (jhorbulyk) [reply Adapters/OpenQuestions.md]: It's not clear to me why **Open** Integration Hub has documentation stored in private repos.

**Pull request merged**